src/deluge/model/mod_controllable/mod_controllable_audio.cpp

```diff
@@ -1307,9 +1307,9 @@ ModelStackWithThreeMainThings* ModControllableAudio::addNoteRowIndexAndStuff(Mod
 	return modelStackWithThreeMainThings;
 }
 
-bool ModControllableAudio::offerReceivedCCToLearnedParams(MIDIDevice* fromDevice, uint8_t channel, uint8_t ccNumber,
-                                                          uint8_t value, ModelStackWithTimelineCounter* modelStack,
-                                                          int32_t noteRowIndex) {
+bool ModControllableAudio::offerReceivedCCToLearnedParams(
```

Comment:
Can we refactor instead of bolting more into this function? This situation with two model stack arguments where one must be null is hard to follow and I think it'll lead to bugs down the road

A custom function just for offering CCs to song level gold knobs might be appropriate

Reply:
Will do! Thx